metadata.json

```diff
@@ -48,18 +48,9 @@
     {
       "operatingsystem": "SLES",
       "operatingsystemrelease": [
-        "10",
-        "11",
         "12"
       ]
     },
-    {
-      "operatingsystem": "OpenSuSE",
-      "operatingsystemrelease": [
-        "13",
-        "42"
-      ]
-    },
```

Comment:
Dropping OpenSuSE all together. Both versions are EoL and I dont feel like adding 15 without tests to verify that it's actually working.